### PR TITLE
TestCafe: add material theme testing for Form

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: .github/workflows/devextreme_npm_tests.yml [hunk 1]
```diff
@@ -283,7 +283,7 @@ jobs:
     - name: Use Node.js
       uses: actions/setup-node@v3
       with:
-        node-version: '15'
+        node-version: '16'
```
Comment (ksercs) [inline .github/workflows/devextreme_npm_tests.yml:286]: we should discuss it with DevOps squad

File: testing/testcafe/tests/form/form.buttonItem.ts [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-import url from '../../../helpers/getPageUrl';
-import createWidget from '../../../helpers/createWidget';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+import { testScreenshot } from '../../helpers/themeUtils';
 
-fixture`Form`
-  .page(url(__dirname, '../../container.html'));
+fixture.disablePageReloads`Form`
+  .page(url(__dirname, '../containerQuill.html'));
```
Comment (ksercs) [inline testing/testcafe/tests/form/form.buttonItem.ts:7]: why quill?
```suggestion
  .page(url(__dirname, '../container.html'));
```

**Review by ksercs — APPROVED** [review]

File: testing/testcafe/tests/form/form.buttonItem.ts (hunk 2 above)
Comment (EugeniyKiyashko) [reply testing/testcafe/tests/form/form.buttonItem.ts:7]: Form uses htmlEditor inside

Comment (ksercs) [reply testing/testcafe/tests/form/form.buttonItem.ts:7]: got it, HtmlEditor is used inside the form

File: .github/workflows/devextreme_npm_tests.yml (hunk 1 above)
Comment (EugeniyKiyashko) [reply .github/workflows/devextreme_npm_tests.yml:286]: I called Timofey for review

**Review by timbset — APPROVED** [review]

**Pull request merged**